specification/2.0/README.md

```diff
@@ -2250,7 +2250,7 @@ The root object for a glTF asset.
 
 Additional properties are allowed.
 
-* **JSON schema**: [gltf.schema.json](schema/gltf.schema.json)
+* **JSON schema**: [glTF.schema.json](schema/glTF.schema.json)
```

Comment:
The properties reference is auto-generated by [Wetzel](https://github.com/AnalyticalGraphicsInc/wetzel).

@BeardedGnome @HowardWolosky We should fix this in the generator.